### PR TITLE
Liftoff

### PR DESCRIPTION
**Pull request by tdido**
### Description

Adding wrapper for liftoff (https://github.com/agshumate/Liftoff/tree/master/liftoff)

### QC

For all wrappers added by this PR, I made sure that

* [ x ] there is a test case which covers any introduced changes,
* [ x ] `input:` and `output:` file paths in the resulting rule can be changed arbitrarily,
* [ x ] all `environment.yaml` specifications follow [the respective best practices](https://stackoverflow.com/a/64594513/2352071),
* [ x ] wherever possible, command line arguments are inferred and set automatically (e.g. based on file extensions in `input:` or `output:`),
* [ x ] all fields of the example rules in the `Snakefile`s and their entries are explained via comments (`input:`/`output:`/`params:` etc.),
* [ x ] `stderr` and/or `stdout` are logged correctly (`log:`), depending on the wrapped tool,
* [ x ] the `meta.yaml` contains a link to the documentation of the respective tool or command,
* [ x ] `Snakefile`s pass the linting (`snakemake --lint`),
* [ x ] `Snakefile`s are formatted with [snakefmt](https://github.com/snakemake/snakefmt),
* [ x ] Python wrapper scripts are formatted with [black](https://black.readthedocs.io).

### CODE REVIEW

File: bio/liftoff/test/Snakefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+rule liftoff:
+    input:
+        ref = "{ref}.fasta.gz",
+        tgt = "{ref}.fasta.gz",
```
Comment (jafors) [inline bio/liftoff/test/Snakefile:4]: This seems to be an unlucky choice for this example Snakefile, as both reference and target fasta are identical here (`{ref}` evaluates to the same wildcard). If you want to use wildcards, use `{ref}` and `{target}` or something like this.

Also, please format the Snakefile using `snakefmt` to get rid of the spaces around the `=` etc.

File: bio/liftoff/environment.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+channels:
+  - bioconda
+  - conda-forge
+dependencies:
+  - liftoff == 1.6.*
```
Comment (jafors) [inline bio/liftoff/environment.yaml:5]:
```suggestion
  - liftoff =1.6
```

**Review by jafors — CHANGES_REQUESTED** [review]
Thanks for this addition! Just a few comments.

File: bio/liftoff/environment.yaml (hunk 2 above)
Comment (tdido) [reply bio/liftoff/environment.yaml]: Thanks for the review! Changes applied.

**Review by jafors — APPROVED** [review]
Thanks for the changes! Looks good.

**Pull request merged**